extensions/amp-ad/0.1/test/test-amp-ad.js

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-import {a4aRegistry} from '../../../../ads/_a4a-config';
+import {createIframePromise} from '../../../../testing/iframe';
```

Comment:
This line is causing an error in the Travis build. Are you able to see the build logs? https://travis-ci.org/ampproject/amphtml/builds/274192851?utm_source=github_status&utm_medium=notification

You can also see that by clicking on "Details" next to the travis row on the main PR view